intranet/templates/itemreg/register_form.html

```diff
@@ -89,9 +89,18 @@ <h2>
     <table>
     {% csrf_token %}
     {{ form.as_table }}
-    <tr><td>&nbsp;</td><td><br><input type="submit"></td></tr>
+    <tr><td>&nbsp;</td><td><br><input type="submit" value="Register Device"></td></tr>
     </table>
 
     </form>
-</div>
-{% endblock %}
```

Comment:
This shouldn't be removed. It's causing your checks to fail.
```
</div>
{% endblock %}
```

Reply:
> This shouldn't be removed. It's causing your checks to fail.

Whoops, I don't know how I removed that. It's fixed now.